src/parser/jobs/rpr/modules/Procs.tsx

```diff
@@ -0,0 +1,92 @@
+import {t} from '@lingui/macro'
+import {Trans, Plural} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Procs as CoreProcs} from 'parser/core/modules/Procs'
+import {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+// these values may need to be changed in future.
+const SEVERITIES = {
+	EXPIRED: {
+		1: SEVERITY.MEDIUM,
+		2: SEVERITY.MAJOR,
+	},
+	OVERWRITTEN: {
+		1: SEVERITY.MINOR,
+		5: SEVERITY.MEDIUM,
+	},
+}
+
+//potency loss due to not using any one proc, enshrouded or reaver.
+const LOST_PROC_POTENCY = 60
+
+export class Procs extends CoreProcs {
+	static override title = t('rpr.enhanced-gibbet-gallows.title')`Incorrect Soul Reaver Usage`
```

Comment:
Not sure about the title here, since it's not actually tracking Soul Reaver stacks as such. That said, I just wanna change the handle to reflect the module name. Removed "incorrect" from the title because it's showing usage, whether it's correct or not.

```suggestion
	static override title = t('rpr.procs.title')`Soul Reaver Usage`
```